pkg/kp/lock_test.go

```diff
@@ -0,0 +1,39 @@
+package kp
+
+import (
+	"testing"
+	"time"
+)
+
+const (
+	lockMessage = "Locked by lock_test.go"
+)
+
+func TestLock(t *testing.T) {
```

Comment:
While I don't want to have fully blown-out test names a la Rspec land, I do think it's valuable to know exactly what is being tested in the form of comments so we can establish how comprehensive coverage is at a glance.